presto-main/src/test/java/com/facebook/presto/execution/TestCreateTableTask.java

```diff
@@ -190,13 +249,29 @@ public Type getType(TypeSignature signature)
 
         public int getCreateTableCallCount()
         {
-            return createTableCallCount.get();
+            return metadatas.size();
+        }
+
+        public List<ConnectorTableMetadata> getReceivedTableMetadata()
+        {
+            return metadatas;
         }
 
         @Override
         public void dropColumn(Session session, TableHandle tableHandle, ColumnHandle column)
         {
             throw new UnsupportedOperationException();
         }
+
+        @Override
+        public Set<ConnectorCapabilities> getConnectorCapabilities(Session session, ConnectorId catalogName)
+        {
+            return connectorCapabilities;
+        }
+
+        public void setConnectorCapabilities(ConnectorCapabilities... connectorCapabilities)
+        {
+            this.connectorCapabilities = immutableEnumSet(ImmutableList.copyOf(connectorCapabilities));
```

Comment:
The copy should not be needed as there is an array variant of `immutableEnumSet`

Reply:
The signature of that method makes this suggestion awkward to implement:

`public static <E extends Enum<E>> ImmutableSet<E> immutableEnumSet(E anElement, E... otherElements) {`